### DISCUSSION TITLE
WSL2 + Docker

### DISCUSSION THREAD

**Discussion by androidformax** [Evil0ctal/Douyin_TikTok_Download_API#260] [Q&A]
When I start docker on wsl2 I'm getting "connection refused".
When I look at docker ps i see that is no have a port and IP 
```
CONTAINER ID   IMAGE                                  COMMAND        CREATED         STATUS         PORTS     NAMES
d109ab8b1117   evil0ctal/douyin_tiktok_download_api   "./start.sh"   5 seconds ago   Up 4 seconds  [NO INFO]  douyin_tiktok_download_api
```
I tried writing in the **docker-compose.yml**
```
ports:
      - 8080:8080
```

**config.ini** 
```
# Default port of API
Port = 8080
```

I restart, but I still get the error
I have other docker configurations working fine though.

I also configured on 8082 but had the same problem.
![image](https://github.com/Evil0ctal/Douyin_TikTok_Download_API/assets/13571444/250846b5-9a0a-4b74-ac22-1511bbefe4c9)

Has anyone encountered a similar problem? Any ideas for a solution?

**Comment by androidformax** [comment]
Here is the ps command response the port is empty. 
![image](https://github.com/Evil0ctal/Douyin_TikTok_Download_API/assets/13571444/5b75b7ec-d9bf-4039-a215-8a636bea5675)

Although everything in the config is correct:
![image](https://github.com/Evil0ctal/Douyin_TikTok_Download_API/assets/13571444/24d13c1b-85e5-4898-9665-95964536762d)
![image](https://github.com/Evil0ctal/Douyin_TikTok_Download_API/assets/13571444/a93c5ddb-c867-4a40-b31c-565d26b01cd8)

**Comment by androidformax** [comment]
Solved the problem. The port should have been specified above, and the host line should have been deleted... After that, everything worked as it should!
![image](https://github.com/Evil0ctal/Douyin_TikTok_Download_API/assets/13571444/296bec83-72b6-47ab-b43a-1dd112460008)

**Reply by Evil0ctal** [reply]
Very nice 😊
